wcfsetup/install/files/lib/system/form/builder/container/wysiwyg/WysiwygFormContainer.class.php

```diff
@@ -448,12 +481,14 @@ public function populate() {
 			$this->setAttachmentHandler();
 		}
 		
-		$this->getDocument()->addButton(
-			WysiwygPreviewFormButton::create($this->getWysiwygId() . 'PreviewButton')
-				->objectType($this->messageObjectType)
-				->wysiwygId($this->getWysiwygId())
-				->objectId($this->getObjectId())
-		);
+		if ($this->enablePreviewButton === true) {
```

Comment:
```suggestion
		if ($this->enablePreviewButton) {
```

Reply:
Why should we remove the `=== true` check here?

The only reason I can imagine is to also support 1 and 0 for valid values (and other equivalents). They will be false in any case in my suggestion, espacially `1 === true` will result to false.

In this case I would suggest to use only `$this->enablePreviewButton == true`, which doesn't take care of the actual datatype , but that will then be equivalent to your suggestion.

Reply:
> Why should we remove the `=== true` check here?

An `if` already carries an implicit `== true` within its definition. Writing an explicit `== true` is redundant.